_includes/components/info-tiles.html

```diff
@@ -19,11 +19,7 @@ <h2>{{ include.heading }}</h2>
     {%- capture attrs %}{{ attrs }} data-color="{{- info_tile.color -}}"{% endcapture -%}
   {%- endif -%}
   <div class="info-tile"{{ attrs }}>
-  {%- if include.heading -%}
-    <h4>{{ info_tile.heading }}</h4>
-  {%- else -%}
     <h3>{{ info_tile.heading }}</h3>
```

Comment:
The h4 wasn't in sequential order

Reply:
Does this always being an h3 make a11y complain if the include didn't have a heading to create the preceding h2?

Reply:
It complains when you jump a level. In this case it jumped from h2 to h4, skipping h3